aries-site/src/pages/extend/developer-guidance.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Anchor } from 'grommet';
+import { CardGrid, Meta, SubsectionText } from '../../components';
+import { ContentSection, Layout, Subsection } from '../../layouts';
+import { getPageDetails, getRelatedContent } from '../../utils';
+
+const title = 'Developer Guidance';
+const topic = 'Extend';
+const pageDetails = getPageDetails(title);
+const relatedContent = getRelatedContent(title);
+const DeveloperGuidance = () => {
+  return (
+    <Layout title={title}>
+      <Meta
+        title={title}
+        description={pageDetails.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/extend/developer-guidance"
+      />
+      <ContentSection>
+        <Subsection name={title} level={1} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+        </Subsection>
+        <Subsection name="Getting started">
+          <SubsectionText>
+            We're excited you're using the HPE Design System! Below are a set of
+            resources that will help you get your application set-up with
+            Grommet and the HPE theme. If you don't find what you're looking
+            for, please reach out in the #hpe-design-system channel on{' '}
+            <Anchor
+              label="Slack"
+              href="https://grommet.slack.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            />
+            .
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Preferred environment" level={3}>
+          <SubsectionText>
+            The HPE Design System is primarily focused on user interfaces
```

Comment:
This may have been addressed before since this was already some copy but are we assuming they know javascript or should we also link to that ? as well as node ?

Reply:
I think by saying we are recommending ReactJS and then providing links to the React documentation that this should take care of that, but let's keep it as a note of something that may be useful to add into the copy later on.

Reply:
Yeah that sounds good. I think there are a few more blogs as well we can go back and put in from the developer portal